steward/src/allocation.rs

```diff
@@ -68,8 +69,7 @@ pub async fn decide_rebalance(tick_range: Vec<somm::TickRange>, cellar_address:
             }
         };
         let config = APP.config();
-        let pair_id = &config.cellars[0].pair_id;
-        let allocation = to_allocation(tick_range, pair_id.to_string(), eth_gas_price.as_u64());
+        let allocation = to_allocation(tick_range, cellar_address.to_string(), eth_gas_price.as_u64());
```

Comment:
So there's a weird issue here with `H160`, `to_string` returns a truncated version of the output rather than the full hex string. This is how it was addressed in the gravity bridge:

https://github.com/PeggyJV/gravity-bridge/blob/4159c4fd56be1bf1ba586698cabeb5c8ca1f46c9/orchestrator/gravity_utils/src/ethereum.rs#L31-L33

Reply:
Another alternative would be to just store the address as a string in `parse_cellar_id` and update `CellarId` to store a string. You could convert it to an `H160` elsewhere if it's needed.